Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.6
+
+# Set up code directory
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+# Install Linux dependencies
+RUN apt-get update && apt-get install -y libssl-dev
+
+# Copy over requirements
+COPY setup.py .
+COPY README.md .
+COPY requirements-dev.txt .
+COPY requirements-docs.txt .
+
+# Install python dependencies
+RUN pip install -r requirements-dev.txt
+RUN pip install -e .
+
+WORKDIR /code
```

Comment:
I think I could get onboard with the inclusion of a dockerfile but would mind extracting it into a separate pull request and writing up some basic documentation in the `README.md` explaining how one might go about using this.